packages/react/src/generators/application/lib/add-cypress.ts

```diff
@@ -6,8 +6,11 @@ export async function addCypress(host: Tree, options: NormalizedSchema) {
   if (options.e2eTestRunner !== 'cypress') {
     return () => {};
   }
-  await ensurePackage(host, '@nrwl/cypress', nxVersion);
-  const { cypressProjectGenerator } = await import('@nrwl/cypress');
+
+  ensurePackage(host, '@nrwl/cypress', nxVersion);
+  const { cypressProjectGenerator } = await import(
+    '@nrwl/cypress/src/generators/cypress-project/cypress-project'
```

Comment:
nit: shouldn't need to deep import. [`cypressProjectGenerator` should be exported from the package top level.](https://github.com/nrwl/nx/blob/master/packages/cypress/index.ts#L1)

Reply:
ok. done. changed.